bossutils/vault.py

```diff
@@ -35,7 +35,7 @@ def catch_expire(function):
     def wrapper(*args, **kwargs):
         try:
             return function(*args, **kwargs)
-        except Exception as e:
+        except Forbidden as e:
```

Comment:
You either need to qualify the class reference or import the class from `hvac`

Reply:
So sorry I updated that but forgot to commit!

Reply:
There we go, sorry! Didn't notice I forgot to commit